src/ray/raylet/worker.cc

```diff
@@ -110,10 +107,16 @@ void Worker::AcquireTaskCpuResources(const ResourceIdSet &cpu_resources) {
   task_resource_ids_.Release(cpu_resources);
 }
 
+void Worker::SetGetTaskReplyAndCallback(
+    rpc::GetTaskReply *reply, const rpc::SendReplyCallback &&send_reply_callback) {
+  RAY_CHECK(reply_ == nullptr && send_reply_callback_ == nullptr);
+  reply_ = reply;
```

Comment:
Hmm actually maybe I'm misunderstanding something, but why do we need to save `reply` along with `send_reply_callback`?

Reply:
Use this callback to send reply to raylet client.